<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Start_Camera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera Ac</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop_Camera">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>490</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera Kapat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Image_Capture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Foto Cek</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start_Recording">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Kayıt Baslat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop_Recording">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Kayıt Durdur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Camera_Devices">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aygıtlar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video_start">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video_stop">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Time">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_first_Minute">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00;00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_second_Minute">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00;00</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_color" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>391</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>21</width>
       <height>391</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_Video">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>391</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	border:1px solid black;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>10</y>
        <width>16</width>
        <height>391</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCameraViewfinder" name="widget_Camera_View" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video_stop_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya_Sec">
    <property name="title">
     <string>Dosya Sec</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuDosya_Sec"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Dosya Gezgini</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header location="global">qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
